app/models/order.rb

```diff
@@ -4,6 +4,10 @@ class Order <ApplicationRecord
   has_many :item_orders
   has_many :items, through: :item_orders
 
+  belongs_to :user, optional: true
```

Comment:
I am not sure we should have an optional qualifier here, shouldn't orders always belong to a user?  I think in the visitor's cart, they are not allowed to checkout and create an order unless they login.